Ships as Raw

```diff
@@ -0,0 +1,860 @@
+
+Name|Gladiator I-class Star Destroyer
+Type|Gladiator
+Keywords|0
+Faction|Imperial
+Size|Small
+Points|56
+Hull|5
+defenseTokens|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0

```

Comment:
Probs want a different way to note an invalid speed, since we have 0 as 0-clicks elsewhere

Reply:
Should I put a -  sign?

Reply:
That sounds good to me.